client/planning-extension/src/extension.ts

```diff
@@ -128,7 +128,9 @@ const extension: IExtension = {
                         },
                     },
                 },
-                notifications: {'email:notification:assignments': {type: 'email'}},
+                notifications: {
+                    'email:notification:assignments': {name: superdesk.localization.gettext('Assignment')}
```

Comment:
Do desktop notifications work for this?

Reply:
Check with Petr if it can be made to send desktop notifications too. Because in settings users can toggle both desktop and emai. And if they set desktop to "on" and don't get notified - that's a bug.

Reply:
It does send email & desktop.

Reply:
It'd be good to update the name then that indicates otherwise.